sklego/mixture.py

```diff
@@ -0,0 +1,94 @@
+import inspect
+
+import numpy as np
+from sklearn.base import BaseEstimator, ClassifierMixin
+from sklearn.mixture import GaussianMixture
+from sklearn.utils import check_X_y
+from sklearn.utils.validation import check_is_fitted, check_array, FLOAT_DTYPES
+
+
+def _check_gmm_keywords(kwargs):
+    for key in kwargs.keys():
+        if key not in inspect.signature(GaussianMixture).parameters.keys():
+            raise ValueError(f"Keyword argument {key} is not in `sklearn.mixture.GaussianMixture`")
+
+
+class GMMClassifier(BaseEstimator, ClassifierMixin):
+    def __init__(self, **gmm_kwargs):
+        self.gmm_kwargs = gmm_kwargs
+        self.classes = None
+        self.gmms = None
+        _check_gmm_keywords(gmm_kwargs)
+
+    def fit(self, X: np.array, y: np.array) -> "GMMClassifier":
+        """
+        Fit the model using X, y as training data.
+
+        :param X: array-like, shape=(n_columns, n_samples,) training data.
+        :param y: array-like, shape=(n_samples,) training data.
+        :return: Returns an instance of self.
+        """
+        X, y = check_X_y(X, y, estimator=self, dtype=FLOAT_DTYPES)
+        self.gmms = {}
+        self.classes = np.unique(y)
+        for cls in self.classes:
```

Comment:
`cls` could be confusing making a dev think it's a classmethod, but I'm not sure whether `cls_` or something would actually be better

Reply:
the letter `c` will do :)